app/src/main/java/org/mozilla/focus/utils/StatusBarUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.mozilla.focus.utils;
+
+import android.view.View;
+import android.view.WindowInsets;
+
+/**
+ * Created by Fer on 08/03/2018.
+ */
+
+public class StatusBarUtils {
+    private static int STATUS_BAR_SIZE = -1;
+
+    public interface StatusBarHeightListener {
+        void onStatusBarHeightFetched(int statusBarHeight);
+    }
+
+    public static void getStatusBarHeight(View view, final StatusBarHeightListener listener) {
+        if (STATUS_BAR_SIZE > 0) {
```

Comment:
\[Checkstyle\] ERROR: 'if' construct must use '\{\}'s\.

Reply:
This seems like an error, because, in fact, there are curly brances on the if statement